ipi_tests/regression_tests/README

```diff
@@ -1,15 +1,33 @@
-See documentaion of how to run the regression tests by running:
+----------------------------------------------------------------------------------
+                   R E G R E S S I O N   T E S T S
+----------------------------------------------------------------------------------
+
+INFORMATION ON CREATING NEW REGRESSION TESTS
+----------------------------------------------------------------------------------
+
+See the documentaion of how to run the regression tests by running:
 
    "python test_run.py -h"
 
 To add a new regression test please provide:
 
    - input.xml (and all the files required by it)
 
-   - driver.txt specifing  "<driver> <address> <port> <unix/inet> " (see existing examples)
+   - driver.txt specifying  "<driver> <address> <port> <unix/inet> " (see existing examples)
+	you should run the driver with the command
+ 
+	"i-pi-driver -m <driver> -h <address> -u -o <possible options>" if unix 
+
+   
+   - file_to_check.txt specifying the files serving as reference with their
+     respective filenames (see existing examples)
```

Comment:
Could you point to an example where the "possible options" flags are actually used? Otherwise one has to look for it along the many reg-test

Reply:
yes, done.